packages/gatsby-plugin-page-creator/src/validate-path.js

```diff
@@ -0,0 +1,21 @@
+const systemPath = require(`path`)
```

Comment:
It'd be nice to keep sharing these internal utility functions with gatsby core

Reply:
Actually this isn't necessary now since we're removing layouts in v2 — so this is the only package using this.